Program.cs

```diff
@@ -202,6 +203,60 @@ public static void Main(string[] args)
                     // repackage for distribution
                     runCommand("ditto", $"-ck --rsrc --keepParent --sequesterRsrc {stagingApp} {zippedApp}");
 
+                    break;
+                
+                case RuntimeInfo.Platform.Linux:
+                    
+                    runCommand("dotnet", $"publish -f netcoreapp3.1 -r linux-x64 {ProjectName} -o {stagingPath}/osu.AppDir/usr/bin/ --configuration Release /p:Version={version} --self-contained");
+                    
+                    // mark output as executable
+                    runCommand("chmod", $"+x {stagingPath}/osu.AppDir/");
+
+                    // create AppRun file

```

Comment:
Rather than creating these files each time, it may be nicer to put them in a zip file, similar to how the macOS build is managed (see `osu!.app-template.zip`).